.github/workflows/raspberrypi3-64.yml

```diff
@@ -0,0 +1,47 @@
+name: Raspberry Pi 3 (using 64bit OS)
+
+on:
+  # With these triggers the Yocto jobs will run
+  # in parallel with the Flowzone jobs, which is fine for now
+  # and allows us to better control what we want to test and when.
+  # It is expected that Flowzone could fail, but yocto jobs will run.
+  pull_request:
+    branches:
+      - "main"
+      - "master"
+  pull_request_target:
+    branches:
+      - "main"
+      - "master"
+  push:
+    tags:
+      - v[0-9]+.[0-9]+.[0-9]+\+?r?e?v?*
+      - v20[0-9][0-9].[0-1]?[1470].[0-9]+
+  workflow_dispatch:
+
+jobs:
+  yocto:
+    name: Yocto
+    # FIXME: This workflow has dependencies on scripts in the balena-yocto-scripts repository
+    # which is pinned separately as a submodule in the device repo. Expect some drift but try to retain compatibility.
+    uses: balena-os/balena-yocto-scripts/.github/workflows/yocto-build-deploy.yml@d59fac4cce1dcff0b423ac97aeccbd7f4486b9c2 # v1.25.8
+    # Prevent duplicate workflow executions for pull_request (PR) and pull_request_target (PRT) events.
+    # Both PR and PRT will be triggered for the same pull request, whether it is internal or from a fork.
+    # This condition will prevent the workflow from running twice for the same pull request while
+    # still allowing it to run for all other event types.
+    if: (github.event.pull_request.head.repo.full_name == github.repository) == (github.event_name == 'pull_request')
+    secrets: inherit
+    with:
+      machine: raspberrypi3-64
+      # Don't deploy any artifacts for RPI build+test sanity workflows
+      deploy-s3: false
+      deploy-hostapp: false
+      deploy-ami: false
```

Comment:
These should be removed and default to true.

Reply:
I was thinking for testing purposes atm, we keep them false. 
Or, we don't merge the PR until we are ready to deploy?

Reply:
if you point to the latest GHA workflow then none of those are deployed for open PRs anyway, might as well remove them